test/language/statements/class/subclass/class-definition-null-proto-missing-return-override.js

```diff
@@ -38,4 +38,4 @@ class Foo extends null {
 
 var foo = new Foo();
```

Comment:
Ah! That's what I missed.

Yes that is my interpretation too. Thanks for the confirmation.